userspace/libscap/engine/gvisor/scap_gvisor.cpp

```diff
@@ -116,6 +116,15 @@ int32_t engine::init(std::string config_path, std::string root_path, bool no_eve
 		m_root_path = root_path;
 	}
 
+	if(epoll_timeout > 0)
```

Comment:
I'm not sure 0 vs 1ms will make a huge difference (except maybe the cpu fans will make your machine completely take off rather than levitate over the floor) but good point :)